code/modules/cargo/delivery/package.dm

```diff
@@ -1,7 +1,18 @@
 /obj/item/cargo_package
 	name = "cargo package"
-	desc = "An Orion Express cargo package, these packages generally find their way to researchers bunkered up on exoplanets. Always pays an extra 2% tip to the courier."
-	desc_info = "You can deliver this package to a delivery site on an exoplanet to get additional funds for the cargo department's account. An additional 2% is added to your account on delivery. Can be loaded into a cargo pack."
+	desc = "\
```

Comment:
No wasted lines

Reply:
well it is a bit odd tbh, would prefer if the first line wasn't just \

Reply:
I think it's fine, and this is a style I used in many files now.
Like this all the text lines are aligned.